### ISSUE TITLE
ignore entities with duplicated shortid and different _id

### ISSUE THREAD

**Issue by bjrmatos** [jsreport/jsreport-import-export#8]
right now after importing a zip that contains an entity with a shortid that already exists results in some confusion in template references. 

## case description:

there are two exports that contain same shortid for the template “Header"

exporting from 1 → https://playground.jsreport.net/w/anon/BkEHf9MNG-5
exporting from 2 → https://playground.jsreport.net/w/anon/BkEHf9MNG-11

**in normal jsreport installation**, importing the two results in that the "Header" from second export does not get inserted (only a warning is throw in console, nothing show in UI) and it results in that the second example can run but with wrong template assigned in pdf-utils (it runs with the "Header" of first example.

**in jo**, importing the two results in that both "Header" templates are inserted, so both examples contains all required files to run, however one of them runs with wrong template being assigned to pdf-utils, since there are two templates "Header" with the same shortid, pdf-utils always reads just one version of the available templates for both examples.

TL;DR is that importing an entity that contains a shortid that already exists is being added to store normally, and that results in some references by shortid (like in pdf-utils) to get confused.

## solution

we should add a warning when importing (at validation of zip) (visible in UI) that described that some entity has a conflict and that it won't be imported. in import logic the entity with the duplicated shortid should be ignored/skipped (not imported at the end). it is important that the condition for detecting this case should be something like "duplicated shortid and with different id", otherwise imports that updates same entities won't work.
